packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -248,10 +247,8 @@ def get_configuration_parameters_with_type_and_default_values(
 
         configuration_parameters.append(f"{parameter_name}: {parameter_type} = {default_value}")
 
-    # Use line separator and spaces to align the parameters appropriately in the generated file.
-    configuration_parameters_with_type_and_value = f",{os.linesep}        ".join(
-        configuration_parameters
-    )
+    # Join the list of configuration parameters into a comma-separated string.
```

Comment:
Probably this comment is redundant now as it simply restates the obvious.

Reply:
Removed the comment.